src/crypto-api.ts

```diff
@@ -263,3 +274,19 @@ export class DeviceVerificationStatus {
 }
 
 export * from "./crypto-api/verification";
+
+/**
+ * The result of a call to {@link CryptoApi.getCrossSigningStatus}
+ */
+export interface CrossSigningStatus {
+    // True if the public master, self signing and user signing keys exist
```

Comment:
these all need to be doc-comments, with `/** */` delimiters. Otherwise they don't appear at https://pr3452--js-sdk-docs-previews.netlify.app/stable/interfaces/crypto.crosssigningstatus !